experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { IEvent } from "@fluidframework/common-definitions";
+import { LoadableObjectClass } from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+interface IFluidContainerEvents extends IEvent {
+    (event: "connected", listener: (clientId: string) => void): void;
+    (event: "dispose" | "disconnected", listener: () => void): void;
+}
+
+interface IFluidContainer {
+    close: Container["close"];
+    closed: Container["closed"];
+    connected: Container["connected"];
+    initialObjects: RootDataObject["initialObjects"];
```

Comment:
Pulling from `RootDataObject` is at least in the same package, but is still not great since it ties internal implementation to public interface that our customers will use.  It makes more sense to just type this normally too.